python/src/energuide/cli.py

```diff
@@ -1,2 +1,29 @@
+import click
+from energuide import database
+from energuide import transform
+
+
+@click.group()
 def main() -> None:
-    print("Hello")
+    pass
+
+
+@main.command()
+@click.option('--username', envvar=database.EnvVariables.username.value, default=database.EnvDefaults.username.value)
+@click.option('--password', envvar=database.EnvVariables.password.value, default=database.EnvDefaults.password.value)
+@click.option('--host', envvar=database.EnvVariables.host.value, default=database.EnvDefaults.host.value)
+@click.option('--port', envvar=database.EnvVariables.port.value, default=database.EnvDefaults.port.value, type=int)
+@click.option('--db_name', envvar=database.EnvVariables.database.value, default=database.EnvDefaults.database.value)
+@click.option('--collection',
+              envvar=database.EnvVariables.collection.value,
+              default=database.EnvDefaults.collection.value)
+@click.option('--filename', type=click.Path(exists=True), required=True)
```

Comment:
Any particular reason you've put all the authentication arguments onto the command as opposed to the group? I ask because this is different than the approach taken with geocoder.

Reply:
Mainly because it's the only subcommand right now, and I'm not sure if future ones will need them. Maybe? Punt until we know.